spec.md

```diff
@@ -204,15 +205,15 @@ to DANE:
   recipient domain.
 
 * Security: DANE offers an advantage against policy-lookup DoS attacks; that is,
-  while a DNSSEC-signed NX response to a DANE lookup authoritatively indicates
+  while a DNSSEC-signed MX response to a DANE lookup authoritatively indicates
```

Comment:
This was meant to be "NXDOMAIN", not "MX." In other words, a DNSSEC-signed "non-existent" response is authoritative. 

Probably should say "NXDOMAIN", though. :)